sli/interpret.h

```diff
@@ -898,6 +905,18 @@ SLIInterpreter::addmodule( void )
 }
 
 inline void
+SLIInterpreter::reset_modules()
+{
+  std::list< SLIModule* >::iterator i;
```

Comment:
I think `i` is a very confusing variable name for a variable representing modules, in particular since we (ab)use `i` as variable name for the interpreter in many places.